server/controllers/devices.coffee

```diff
@@ -61,7 +61,7 @@ deviceExists = (login, cb) ->
         else if body.length is 0
             cb null, false
         else
-            cb null, body[0]
+            cb null, body[0]?.value
```

Comment:
@aenario I have no idea what have changed that causes this. Does it ring a bell for you?